healthtools_ke_api/__init__.py

```diff
@@ -31,7 +35,9 @@ def index():
             "/": {"methods": ["GET"]},
             "/nurses": {"methods": ["GET"]},
             "/doctors": {"methods": ["GET"]},
-            "/clinical-officers": {"methods": ["GET"]}
+            "/clinical-officers": {"methods": ["GET"]},
+            "/health_facilities": {"methods": ["GET"]}
```

Comment:
`health-facilities`

Let's try be more careful on following any of the other URLs / conventions already in here.